src/components/Images.vue

```diff
@@ -234,28 +235,33 @@ export default {
     },
     deleteImage(obj) {
       this.kimRunningCommand = `delete ${ obj.imageName }:${ obj.tag }`;
+      this.refreshImagesOnEnd = true;
       this.startRunningCommand('delete');
-      ipcRenderer.send('confirm-do-image-deletion', obj.imageName, obj.imageID);
+      ipcRenderer.send('confirm-do-image-deletion', obj.imageName.trim(), obj.imageID.trim());
     },
     doPush(obj) {
       this.kimRunningCommand = `push ${ obj.imageName }:${ obj.tag }`;
+      this.refreshImagesOnEnd = false;
       this.startRunningCommand('push');
-      ipcRenderer.send('do-image-push', obj.imageName, obj.imageID, obj.tag);
+      ipcRenderer.send('do-image-push', obj.imageName.trim(), obj.imageID.trim(), obj.tag.trim());
```

Comment:
Sounds like using `ipcRenderer.invoke(…).then(…)` would be a lot cleaner?

Reply:
For a UI, I prefer the current model of sending an event async, and then having another handler deal with any reply.